frontend/src/locales/scripts/en.json5

```
@@ -1051,6 +1051,37 @@
       creator: "Creator",
       source: "Source",
     },
+    pageTitle: {
+      tag: {
+        /**
+         * {tag} will be a dynamic value such as "cat". We cannot change its case or form.
+         * You can change the sentence to add more context and make the sentence
+         * grammatically correct, for instance, to "Images with the tag {tag}".
```

Should this be media specific, like the source example below?

```suggestion
         * grammatically correct, for instance, to "Audio tracks with the tag {tag}".
```

---

Yes! Thank you